.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,25 @@
+## Pelorus <VERSION> release
+
+<!-- Write this section if Pull Request has linked issues -->
+## Linked Issues
+
+<!-- Use this if merging should auto-close an issue, one issue per line -->
+resolves #<!-- issue number -->
+
+<!-- Use this if merging should NOT auto-close an issue, one issue per line -->
+related to #<!-- issue number -->
+
+## Requirements for release
+
+- [ ] Update and review GitHub release body in https://github.com/dora-metrics/pelorus/releases
+- [ ] Test Pelorus Operator
+- [ ] Pelorus Operator in OpenShift Marketplace
+- [ ] GitHub release
+- [ ] Send message notification to
+
+    > (ALERT COMMAND) 🎉🎉 Pelorus Operator v<VERSION> is now available in the OpenShift Marketplace 🎉🎉
+    >
+    >Kindly take a moment to review the release notes which can be found at https://github.com/dora-metrics/pelorus/releases/tag/v<VERSION>
+
+    - [ ] Internal Red Hat pelorus Slack channel
```

Comment:
Not good idea to add URL/emails direct here, right?

Reply:
Yeah I would not mention that, this is still community project, so not only Red Hat may be interested.